.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Task/Issue URL:
```

Comment:
`native-github-asana-sync` needs a trigger phrase so I’ve created a pull request template to encourage it.